odl/solvers/functional/default_functionals.py

```diff
@@ -2226,8 +2226,10 @@ def __init__(self, space, diameter=1, sum_rtol=None):
             Domain of the functional.
         diameter : positive float, optional
             Diameter `r` of simplex.
-        sum_rtol : float
-            Relative tolerance for sum comparison.
+        sum_rtol : float, optional
+            Relative tolerance for sum comparison. If set to None, the default
+            is ``space.size`` times ``1e-10`` when ``space.dtype`` is
+            ``float64`` and ``1e-6`` otherwise.
```

Comment:
This becomes almost a bit too much backticked stuff mixed with regular English. I think this is a bit easier to comprehend:
```
Default:

- ``space.dtype == 'float64'``: ``1e-10 * space.size``
- Otherwise: ``1e-6 * space.size``
```